03week/towersOfHanoi.js

```diff
@@ -13,30 +13,67 @@ let stacks = {
   c: []
 };
 
+
```

Comment:
3 Tests did not pass.
Missing readme code plan file
Did not see additional 3 tests added.